src/views/FigureComponent.tsx

```diff
@@ -21,6 +21,7 @@ import {
   UnsupportedFormatFileIcon,
 } from '@manuscripts/style-guide'
 import { FigureNode } from '@manuscripts/transform'
+import { ProsemirrorNode } from '@remirror/pm'
```

Comment:
this type has to come from prosemirror-model as alias:
import { Node as ProsemirrorNode } from 'prosemirror-model'

Reply:
updated